app/repositories/FileNotificationRepository.scala

```diff
@@ -91,7 +91,14 @@ class FileNotificationRepository @Inject()(mongoComponent: MongoComponent,
     collection.find(in("status", Seq(
       RecordStatusEnum.PENDING.toString,
       RecordStatusEnum.NOT_PROCESSED_PENDING_RETRY.toString,
-      RecordStatusEnum.FAILED_PENDING_RETRY.toString): _*
+      RecordStatusEnum.FAILED_PENDING_RETRY.toString,
+      RecordStatusEnum.FILE_NOT_RECEIVED_IN_SDES_PENDING_RETRY.toString): _*
+    )).toFuture()
+  }
+
+  def getNotificationsSentSDES(): Future[Seq[SDESNotificationRecord]] = {
```

Comment:
Could this method be made generic so it takes a status as a parameter?
```suggestion
  def getNotificationsByStatus(status: RecordStatusEnum.Value): Future[Seq[SDESNotificationRecord]] = {
```